onnxruntime/python/tools/bert/OnnxModel.py

```diff
@@ -267,7 +270,29 @@ def convert_model_float32_to_float16(self):
                 for att in node.attribute:
                     if att.name == 'to' and att.i == 1:
                         att.CopyFrom(onnx.helper.make_attribute("to", 10))
-   
+
+        for input_value_info in graph.input:
+            if input_value_info.type.tensor_type.elem_type == 1:
```

Comment:
Better to use some constant (like TensorProto.FLOAT and TensorProto.FLOAT16) instead of 1 and 10.